content/en/blog/2023/end-user-q-and-a-02.md

```diff
@@ -265,8 +265,9 @@ to Jaeger.
 
 ## What's next?
 
-If you’d like to see my conversation with Doug in full, you can check out the
-video [here](https://www.youtube.com/watch?v=ptYWBF-R1Fc).
+If you’d like to see my conversation with Doug in full, there is a
+[private video](https://www.youtube.com/watch?v=ptYWBF-R1Fc) -- you’ll have to
+ask someone for permission to be able to watch it.
```

Comment:
@open-telemetry/docs-approvers Do we know shy this video is not public anymore?

Reply:
@avillela, this was your blog post from about two years ago, and the video is no longer available. Should we remove this paragraph?

Reply:
it states that you have to ask for permissions to watch it, so I assume it is still there but private?

Reply:
@avillela - can you make the video public again?
If not we should remove the link. I'm not seeing any comment that allows me to ask for permission.

Reply:
@chalin @puckpuck @svrnm looks like the video was made private when the main channel admin made all the meeting videos private. I think this is from before the days of making the videos more polished. I changed it to an unlisted link so that it is accessible.

Reply:
Yes, it is accessible now. Thanks @avillela!  🙌🏻